newIDE/app/src/UI/Theme/DefaultTheme/index.js

```diff
@@ -1,5 +1,8 @@
 // @flow
 import { createMuiTheme, darken, lighten } from '@material-ui/core/styles';
+import { compose } from '../../../Utils/Object';
+import { text, formula as i18n } from '../i18n';
```

Comment:
as mentioned in other comments, let's entirely remove these and use a function `convertToRtl` that takes a muiTheme and returns the rtl-enabled muiTheme. We then use it in getTheme, and we should hopefully be done.